sparkmagic/sparkmagic/utils/configuration.py

```diff
@@ -271,6 +271,13 @@ def kerberos_auth_configuration():
         "mutual_authentication": REQUIRED
     }
 
+@_with_override
+def progress_indicator_class():
+    return 'sparkmagic.utils.progress.defaultProgressIndicator'
+
+@_with_override
+def startup_info_display_class():
+    return 'sparkmagic.utils.startupinfo.defaultStartupInfoDisplay'
```

Comment:
Can you add unit tests that will tell us if these paths point to a class that doesn't exist? This will be helpful for future refactors